includes/class-gp-translation-helpers.php

```diff
@@ -403,7 +403,7 @@ function( $reason ) use ( $all_comment_reasons ) {
 		$comment              = sanitize_text_field( $_POST['data']['comment'] );
 
 		if ( ! $locale_slug || ! $translation_id_array || ! $original_id_array || ( ! $comment_reason && ! $comment ) ) {
-			wp_send_json_error();
+			wp_send_json_error( 'Oops! One or more required parameters missing' );
```

Comment:
Could we split this up into multiple `if`s so that the error message can be more meaningful.